### PR TITLE
Adding new ETA details for SG API

### PR DESCRIPTION
**Pull request by cdwinsnes**

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/NGVhHXNY](https://trello.com/c/NGVhHXNY)

File: content/api/shipping-guide_2/api.raml [hunk 1]
```diff
@@ -281,9 +281,64 @@ documentation:
...
+              </ns2:TimeSlots>
+          </ns2:ExpectedDeliveryDate>
+          <ns2:AlternativeDeliveryDates/>
+      </ns2:ExpectedDelivery>
+      ```
+
+      #### How to present the time window data in your checkout
+      We recommend to insert the start (X) and end (Y) time values into the following sentence:
+
+      **Pakken ankommer vanligvis mellom kl. X og Y / The parcel usually arrives between X and Y.**
```
Comment (tomgag) [inline content/api/shipping-guide_2/api.raml:330]: Could we distinguish a bit better between the languages here?
- NO:
- EN:
or something like that

File: content/api/shipping-guide_2/api.raml [hunk 2]
```diff
@@ -281,9 +281,64 @@ documentation:
...
+
+      Together with the expected delivery date, we recommend the following implementation:
+
+      **Forventet levert XX.XX.20XX. Pakken ankommer vanligvis mellom kl. X og Y / Expected delivered XX.XX.20XX. The parcel usually arrives between X and Y.**
```
Comment (tomgag) [inline content/api/shipping-guide_2/api.raml:334]: Similar here if this suggestion is useful

File: content/api/shipping-guide_2/api.raml [hunk 3]
```diff
@@ -281,9 +281,64 @@ documentation:
...
+
+      Event though the API response schema supports minute granularity for start and end time, the minute value will always be set to 0. It is therefore sufficient to parse hours only.
```
Comment (tomgag) [inline content/api/shipping-guide_2/api.raml:336]: typo: Even though

**Review by best-ankitagarwal — APPROVED** [review]
Looks good 👍

**Review by tomgag — APPROVED** [review]
LGTM, a few suggestions only. 👍

File: content/api/shipping-guide_2/api.raml (hunk 3 above)
Comment (cdwinsnes) [reply content/api/shipping-guide_2/api.raml:336]: 👍

**Pull request merged**